ui/pages/core/hyperliquid-referral-consent/hyperliquid-referral-consent.tsx

```diff
@@ -0,0 +1,136 @@
+import React, { useState } from 'react';
+import {
+  Box,
+  Button,
```

Comment:
I think for this design we can now use those from metamask/design-system-react

Reply:
Good shout! I've updated it